assets/css/card.scss

```diff
@@ -0,0 +1,116 @@
+.dx-card {
+    display: flex;
+    position: relative;
+    padding: 2.25rem 1.5rem 1.5rem;
+    margin: 2.5rem 0 1rem;
+    flex-direction: column;
+    border-radius: 4px;
+    box-shadow: 1px 2px 10px rgba(0, 0, 0, 0.1);
+    color: $brown;
+    text-decoration: none;
+
+    &:hover,
+    &:focus {
+        box-shadow: 1px 2px 20px rgba(0, 0, 0, 0.15);
+        color: $brown;
+
+        &:after {
+            position: absolute;
+            bottom: 0;
+            left: 0;
+            content: "";
+            width: 100%;
+            border-bottom-left-radius: 4px;
+            border-bottom-right-radius: 4px;
+            border-bottom: 4px solid $yellow;
+        }
+    }
+
+    > i {
+        color: $brown;
+    }
+    
+
+    .dx-card-icon {
+        display: flex;
+        position: absolute;
+        top: -1.25rem;
+        left: 2rem;
+        width: 2.5rem;
+        height: 2.5rem;
+        border-radius: 4px;
+        align-items: center;
+        justify-content: center;
+        color: $brown;
+        background-color: $yellow;
+    }
+
+    .dx-card-content {
+        display: flex;
+        flex-direction: column;
+        margin-bottom: 1.25rem;
+
+        h4 {
+            margin-top: 0;
+        }
+    }
+
+    &.dx-card-horizontal {
+        flex-direction: row;
+        justify-content: space-between;
+        align-items: center;
+        padding-top: 1.5rem;
+
+        .dx-card-icon {
+            font-family: "Akkurat Mono", monospace;
+        }
+        
+        .dx-card-content {
+            margin-bottom: 0;
+            margin-top: 0.75rem;
+
+            h3 {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 spaces (indentation)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5681062)